packages/i18n-utils/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	preset: '../../test/packages/jest-preset.js',
+	rootDir: __dirname,
```

Comment:
I think you don't need `rootDir` or `cacheDirectory`, they come from the `preset`